### DISCUSSION TITLE
如何加载chatgpt example训练的模型，并预测出效果

### DISCUSSION THREAD

**Discussion by cxj01** [hpcaitech/ColossalAI#2870] [Community | Q&A]
目前ChatGPT demo可以使用example的数据集进行训练，但是训练好的模型应该如何加载运行？以及对应的效果应该展示成啥样呢？

**Comment by binmakeswell** [comment]
https://github.com/hpcaitech/ColossalAI/tree/main/applications/ChatGPT#how-to-saveload-checkpoint
Hi @cxj01 May be you can refer it. Thanks.
